docs/developer-guide/3-desktop-version-cheat-sheet.md

```diff
@@ -0,0 +1,59 @@
+# Developer Guide: Desktop Version  
```

Comment:
You'll need to add an entry in `mkdocs.yml` to make it appear in the actual documentation (you can test it with `mkdocs serve`)